package.json

```diff
@@ -42,31 +42,48 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format": "prettier . --write",
+    "lint": "eslint src/"
   },
   "eslintConfig": {
     "extends": [
       "react-app",
       "eslint:recommended",
       "plugin:react/recommended",
       "plugin:@typescript-eslint/recommended",
-      "plugin:jsx-a11y/strict"
+      "plugin:jsx-a11y/strict",
+      "plugin:prettier/recommended"
     ],
     "plugins": [
       "react-hooks",
       "@typescript-eslint",
       "jsx-a11y"
     ],
     "rules": {
+      "no-case-declarations": "off",
       "no-prototype-builtins": "off",
       "prefer-const": "warn",
+      "react/prop-types": "off",
       "react-hooks/rules-of-hooks": "error",
       "react-hooks/exhaustive-deps": "warn",
       "@typescript-eslint/explicit-function-return-type": "off",
       "@typescript-eslint/no-empty-interface": "warn",
+      "@typescript-eslint/no-explicit-any": "warn",
```

Comment:
This is actually not as bad as it is in the frontend. It's only happening in 7 places, so instead of showing warnings for this rule, I would simply go and add a rule ignore comment above the offender lines.